config/settings/dev.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+import dj_database_url
+
+from .base import *  # noqa F403
+
+DEBUG = True
+
+# Base de données :
+# https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+
+if DATABASE_URL := os.environ.get("DATABASE_URL"):
```

Comment:
DATABASE_URL est déjà défini via la variable d'environnement dans `base.py`.

```suggestion
if DATABASE_URL:
```

Comme tu as fait dans `prod.py`.